api/src/main/java/org/openmrs/Encounter.java

```diff
@@ -353,6 +355,36 @@ public void removeObs(Obs observation) {
 	public void setOrders(Set<Order> orders) {
 		this.orders = orders;
 	}
+
+	/**
+	 * @return Returns the order groups
```

Comment:
By convention we don't add @since to things that are not part of the public API which of course applies to new private fields and methods, it is the setters and getters for the fields that should have them since they are public